_posts/newsletter/2021-08-12-this-month-in-solid.md

```diff
@@ -6,23 +6,37 @@ tags: [monthly, updates]
 categories: [Updates]
 ---
 
-## Updates
+### Solid World
+**05 August 2021** 
 
-### [Events](https://solidproject.org/events)
-**05 August 2021** – If there were words to describe August's Solid World, they were – growth, collaboration, and progress. It was time to look back at all the work we’ve done since the beginning of the year. It’s safe to say we made a lot of progress! Different [community members gave us updates](https://vimeo.com/585006498) of their goals and the work they’ve completed so far.  
+If there were words to describe August's Solid World, they were – growth, collaboration, and progress. It was time to look back at all the work we’ve done since the beginning of the year. It’s safe to say we made a lot of progress! Different [community members gave us updates](https://vimeo.com/585006498) of their goals and the work they’ve completed so far.  
+
+#### Updates
 
 Before the presentations kicked off, we got updates from Ruben Verborgh and Oz Olivo. Ruben had exceptional news to tell us. After a year of hard work, he has released [Community Solid Server (CSS) 1.0](solidproject.org/self-hosting/css)!  If you’re a developer looking for a companion server to help build your app, or a  researcher exploring the frontier of Solid, try building on CSS. 
 
 As the CSS progresses, Oz reminded us that Inrupt’s Enterprise Solid Server is also on track, with the next major release slated for this fall. Also, Inrupt recently composed the [Solid Application Compatibility Survey](https://www.research.net/r/solidappcompatibility), and they are looking to hear feedback from any app developers. With that said, we luckily got to listen to assessments from our presenters.
 
+#### Presentations
+
+##### SolidProject Updates: Justin Bingham
+
 Our first presenter was Justin Bingham, the CTO at Janeiro and Solid Team Member. Over the last few months, the Solid Team assigned one of the team members as the project manager to help with the different projects and initiatives. Since the change, Justin leads us through the various tasks they were able to complete. First was migrating the Community Forum to Discourse.org. The second was changing the process of the Solid Application Listing by making it a more collaborative review and submission [by having it on the Forum](https://forum.solidproject.org/c/applications/41). Their latest work is on making substantial updates to the [solidproject.org website](solidproject.org). We all will be excited to see how the changes go!
```

Comment:
```suggestion
Our first presenter was Justin Bingham, the CTO at Janeiro Digital and Solid Team Member. Over the last few months, the Solid Team assigned one of the team members as the project manager to help with the different projects and initiatives. Since the change, Justin leads us through the various tasks they were able to complete. First was migrating the Community Forum to Discourse.org. The second was changing the process of the Solid Application Listing by making it a more collaborative review and submission [by having it on the Forum](https://forum.solidproject.org/c/applications/41). Their latest work is on making substantial updates to the [solidproject.org website](solidproject.org). We all will be excited to see how the changes go!
```